src/main/java/github/scarsz/discordsrv/api/events/VentureChatMessagePostProcessEvent.java

```diff
@@ -0,0 +1,49 @@
+/*-
+ * LICENSE
+ * DiscordSRV
+ * -------------
+ * Copyright (C) 2016 - 2021 Austin "Scarsz" Shapiro
+ * -------------
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * END
+ */
+
+package github.scarsz.discordsrv.api.events;
+
+import org.bukkit.event.Cancellable;
+
+import lombok.Getter;
+import lombok.Setter;
+import mineverse.Aust1n46.chat.api.events.VentureChatEvent;
+
+/**
+ * <p>Called after DiscordSRV has processed a VentureChat message but before being sent to Discord.
+ * Modification is allow and will effect the message sent to Discord.</p>
+ */
```

Comment:
This should specify that it's a VentureChat Bungee message when the VentureChatBungee config option and not just any VentureChat message (something like "... VentureChat bungee message (when the VentureChatBungee config option is enabled) ...") is enabled. 

I also think the regular -PreProcess and -PostProcess events should have a note about these new events (something like "VentureChat bungee messages (when the VentureChatBungee config option is enabled) will use the VentureChatMessage...ProcessEvent instead due to the lack of the Player object")